### PR TITLE
rm: boards: am62xx-evk: update flashing instructions

### PR DESCRIPTION
**Pull request by igoropaniuk**
Update flashing instructions, as new flash.sh script is used for flash automation.

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for ce210be14caca7154b96c389a813f2075b9db4d4 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2375/docs/artifacts/html/index.html

**Review by kprosise — APPROVED** [review]
LGTM

File: source/reference-manual/boards/am62xx-sk.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ Texas Instruments AM62x SKEVM
 Hardware Preparation
 --------------------
 
-Set up the board for booting from MMC1 - SDCard:
+Set up the board for booting from USD DFU mode:
```
Comment (MrCry0) [inline source/reference-manual/boards/am62xx-sk.rst:11]: `s/USD/USB/` ?

File: source/reference-manual/boards/am62xx-sk.rst [hunk 2]
```diff
@@ -18,26 +18,51 @@ Set up the board for booting from MMC1 - SDCard:
...
+
+.. prompt:: bash host:~$, auto
+
+	host:$ ./ti-mfgtool-files-am62xx-evm/flash.sh --wic lmp-factory-image-am62xx-evm.wic
+
+    Load U-Boot via DFU...
+    ------------------------------------------
+      DFU BOOT TIBOOT3: TIFS and R5
+    ------------------------------------------
+    dfu-util: can't detach
```
Comment (angolini) [inline source/reference-manual/boards/am62xx-sk.rst:39]: I have no idea why can't have a red box:
image

**Review by angolini — APPROVED** [review]
LGTM, only minor suggestions

File: source/reference-manual/boards/am62xx-sk.rst [hunk 3]
```diff
@@ -18,26 +18,51 @@ Set up the board for booting from MMC1 - SDCard:
...
+    854511616 bytes (855 MB, 815 MiB) copied, 57,7472 s, 14,8 MB/s
+    Flashing is finished
+
+2. Power off the board.
+
+Configure SD Boot
+-----------------
+
+#. Put the am62xx-sk into boot from SDCard Mode,
+   changing boot switches to: ``01000000 11000010``.
```
Comment (Tim-Anderson) [inline source/reference-manual/boards/am62xx-sk.rst:66]: Should't the flashing be into the eMMC and not an SD card?

**Review by Tim-Anderson — CHANGES_REQUESTED** [review]
I think the point of using DFU was to flash to eMMC not to an SD card.

File: source/reference-manual/boards/am62xx-sk.rst (hunk 3 above)
Comment (igoropaniuk) [reply source/reference-manual/boards/am62xx-sk.rst:66]: nice catch, actually script is flashing to eMMC card (I confused with `stm32-mfgtool-files` for `stm32mp15-disco`). Rewriting the article

File: source/reference-manual/boards/am62xx-sk.rst [hunk 4]
```diff
@@ -18,26 +18,51 @@ Set up the board for booting from MMC1 - SDCard:
...
-
-.. include:: generic-flashing.rst
-
-.. figure:: /_static/boards/am62xx-sk-bottom.png
-     :width: 600
-     :align: center
-
-     SDCard location
+#. Power on the board and flash the ``lmp-factory-image-am62xx-evm.wic``
+   using ``flash.sh`` script from ``ti-mfgtool-files-am62xx-evm`` directory.
```
[the remark this answers is not in the document]
Comment (igoropaniuk) [reply source/reference-manual/boards/am62xx-sk.rst:28]: fixed

File: source/reference-manual/boards/am62xx-sk-prepare.rst [hunk 5]
```diff
@@ -36,14 +36,34 @@ Download necessary files from ``https://app.foundries.io/factories/<factory>/tar
...
+   E.g: ``ti-mfgtool-files-am62xx-evm.tar.gz``
+
+#. Download and extract the file ``ti-mfgtool-files-am62xx-evm.tar.gz``::
+
+      tar -zxvf ti-mfgtool-files-am62xx-evm.tar.gz
+
+#. Organize all the files like the tree below::
+
+      ├── lmp-factory-image-am62xx-evm.wic
+      ├── lmp-factory-image-am62xx-evm.wic.gz
```
[the remark this answers is not in the document]
Comment (igoropaniuk) [reply source/reference-manual/boards/am62xx-sk-prepare.rst]: fixed

File: source/reference-manual/boards/am62xx-sk.rst (hunk 3 above)
Comment (igoropaniuk) [reply source/reference-manual/boards/am62xx-sk.rst:66]: fixed

File: source/reference-manual/boards/am62xx-sk.rst (hunk 1 above)
Comment (igoropaniuk) [reply source/reference-manual/boards/am62xx-sk.rst:11]: fixed

**Comment by doanac** [comment]
Docs for 35d3dd7fb7367fb6e063e858b7261d01e8dc42c0 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2380/docs/artifacts/html/index.html

**Review by Tim-Anderson — APPROVED** [review]
LGTM

**Comment by igoropaniuk** [comment]
@kprosise I think we can merge it, thanks

**Pull request merged**